src/logger.cpp

```diff
@@ -0,0 +1,27 @@
+#include <nomagic_abb_driver/Logger.hpp>
+
+#include <boost/log/utility/setup/console.hpp>
+#include <boost/smart_ptr/shared_ptr.hpp>
+#include <boost/smart_ptr/make_shared_object.hpp>
+#include <boost/log/utility/setup/common_attributes.hpp>
+
+
+namespace abb :: rws :: loggin
```

Comment:
```suggestion
namespace abb :: rws :: logging
```

Reply:
Yes I realized a little late, after I had pushed it. I fixed it now